client/k8s/error.go

```diff
@@ -0,0 +1,30 @@
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/juju/errgo"
+)
+
+var (
+	maskAny = errgo.MaskFunc(errgo.Any)
+)
+
+func maskAnyf(err error, f string, v ...interface{}) error {
```

Comment:
I think I'm OK. I'll look again at kubernetesd and fix but I'll shout if I get stuck.